doc/api/stream.md

```diff
@@ -1222,6 +1222,17 @@ added: v11.4.0
 Is `true` if it is safe to call [`readable.read()`][stream-read], which means
 the stream has not been destroyed or emitted `'error'` or `'end'`.
 
+##### `readable.readableDidRead`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+Allows determining if the stream has been or is about to be read.
+Returns true if `'data'`, `'end'`, `'error'` or `'close'` has been
+emitted.
```

Comment:
What about if/when `'readable'` is emitted or `read()` has been called?

Reply:
> What about if/when 'readable' is emitted

That doesn't mean anything in terms of whether data has been read.

> read() has been called?

`read()` will always emit `'data'` when data is read. Regardless whether there is listener or not.

Reply:
I think it makes sense to include that point in the docs here